src/psm/src/ir_solver.cpp

```diff
@@ -1504,4 +1513,30 @@ bool IRSolver::buildConnection()
   result_ = res;
   return result_;
 }
+
+double IRSolver::getResistance(odb::dbTechLayer* layer) const
+{
+  double res;
+
+  if (layer->getRoutingLevel() == 0) {
```

Comment:
Why the special case for this?

Reply:
For vias, resizer records that layer as the resistance of a via, while the routing layers get recorded as ohm/meter (and PSM needs the vias resistance, so no change ot that, but the routing layers it needs the sheet resistance of the metal).

Reply:
That makes sense - a little comment might be useful for others.